src/main/java/com/powsybl/openloadflow/network/impl/LfStarBus.java

```diff
@@ -21,7 +21,7 @@
     public LfStarBus(ThreeWindingsTransformer t3wt) {
         super(Networks.getPropertyV(t3wt), Networks.getPropertyAngle(t3wt));
         this.t3wt = t3wt;
-        nominalV = t3wt.getLeg1().getTerminal().getVoltageLevel().getNominalV();
+        nominalV = t3wt.getRatedU0();
```

Comment:
You are right we need this to be consistent with the other part of the code. But maybe we should rethink about this as it is not consistant with the way we per unit for the other node. Could we have a bu here with a missing fixed ratio (ratedU1/ratedU0)?

Reply:
Maybe. The default modeling of three windings transformer is ratedU0=ratedU1 but it is configurable.